diag_manager/fms_diag_output_buffer.F90

```diff
@@ -43,12 +43,14 @@ module fms_diag_output_buffer_mod
   integer, allocatable, public :: num_elements(:) !< used in time-averaging
   class(*), allocatable, public :: count_0d(:) !< used in time-averaging along with
                                        !! counter which is stored in the child types (bufferNd)
+                                       !! Always allocated to a real type since its used for weights
```

Comment:
I think this should say "Always allocated to a real type because it is used in a calculation with the variable weights".  I think that's more descriptive